one/src/startup.rs

```diff
@@ -0,0 +1,186 @@
+//! Startup utilities for the Ceramic One daemon.
+
+use anyhow::{anyhow, Result};
+use ceramic_api::InterestService as InterestServiceTrait;
+use ceramic_core::{EventId, Interest, NodeId, StreamId};
+use ceramic_interest_svc::InterestService;
+use std::str::FromStr;
+use std::sync::Arc;
+use tracing::{debug, info, warn};
+
+/// Process initial interests by registering them with the interest service
+pub async fn process_initial_interests(
+    initial_interests: &[String],
+    interest_svc: &Arc<InterestService>,
+    network: &ceramic_core::Network,
+    node_id: &NodeId,
+) -> Result<()> {
+    if initial_interests.is_empty() {
+        return Ok(());
+    }
+
+    info!("Processing {} initial interests", initial_interests.len());
+
+    for stream_id_str in initial_interests {
+        let stream_id_str = stream_id_str.trim();
+        if stream_id_str.is_empty() {
+            continue;
+        }
+
+        // Validate that the model stream ID is parseable
+        let _stream_id = StreamId::from_str(stream_id_str)
+            .map_err(|e| anyhow!("Invalid model ID '{}': {}", stream_id_str, e))?;
+
+        // Create an interest for the "model" separator key covering the full range for this stream
+        // This follows the same pattern as the API endpoint /ceramic/interests/model/{stream_id}
+        let stream_id_bytes = multibase::decode(stream_id_str)
+            .map_err(|e| anyhow!("Failed to decode stream ID '{}': {}", stream_id_str, e))?
+            .1;
+        let start = EventId::builder()
```

Comment:
I was considering if it'd be worth putting this in interest service or core so we could share most of this with the http server. But I think this API is fine to be exposed, I just couldn't remember how to build interest ranges and wanted a higher level API that's like `subscribe_to_model` 😅